frontend/src/scenes/notebooks/Notebook/NotebookPopover.tsx

```diff
@@ -53,8 +54,16 @@ export function NotebookPopoverCard(): JSX.Element | null {
                         to={urls.notebook(selectedNotebook)}
                         onClick={() => setVisibility('hidden')}
                         status="primary-alt"
-                        icon={<IconLink />}
-                        tooltip="Go to Notebook"
+                        icon={<IconOpenInNew />}
+                        tooltip="View notebook outside of popover"
```

Comment:
Feels strange to tell you what this doesn't do, rather than what it will. I say that not knowing what is better - `Dock notebook` is the best I can think of and that is worse imo

Reply:
Yeah. I don't want to get too stuck on it for now